tests/defaults.spec.ts

```diff
@@ -0,0 +1,180 @@
+			m.chai.expect(url.parse(setting).protocol).to.not.exist;
+		});
+	});
+
+	describe('.imageMakerUrl', () => {
+		it('should be a valid url', () => {
+			const setting = utils.evaluateSetting<string>(defaults, 'imageMakerUrl');
+			m.chai.expect(() => url.parse(setting)).to.not.throw(Error);
+		});
+
+		it('should contain an https protocol', () => {
+			const setting = utils.evaluateSetting<string>(defaults, 'imageMakerUrl');
+			m.chai.expect(url.parse(setting).protocol).to.equal('https:');
+		});
+	});
+
+	describe('.deltaUrl', () => {
+		it('should be a valid url', () => {
+			const setting = utils.evaluateSetting<string>(defaults, 'deltaUrl');
+			m.chai.expect(() => url.parse(setting)).to.not.throw(Error);
+		});
+
+		it('should contain an https protocol', () => {
+			const setting = utils.evaluateSetting<string>(defaults, 'deltaUrl');
+			m.chai.expect(url.parse(setting).protocol).to.equal('https:');
+		});
+	});
+
+	describe('.dashboardUrl', () => {
+		it('should be a valid url', () => {
+			const setting = utils.evaluateSetting<string>(defaults, 'dashboardUrl');
+			m.chai.expect(() => url.parse(setting)).to.not.throw(Error);
+		});
+
+		it('should contain an https protocol', () => {
+			const setting = utils.evaluateSetting<string>(defaults, 'dashboardUrl');
+			m.chai.expect(url.parse(setting).protocol).to.equal('https:');
+		});
+	});
+
+	describe('.proxyUrl', () => {
+		it('should be a valid url', () => {
+			const setting = utils.evaluateSetting<string>(defaults, 'proxyUrl');
+			m.chai.expect(() => url.parse(setting)).to.not.throw(Error);
+		});
+
+		it('should not contain a protocol', () => {
+			const setting = utils.evaluateSetting<string>(defaults, 'proxyUrl');
+			m.chai.expect(url.parse(setting).protocol).to.not.exist;
+		});
+	});
+
+	describe('.dataDirectory', () =>
+		it('should be an absolute path', () => {
+			const setting = utils.evaluateSetting<string>(defaults, 'dataDirectory');
+			const isAbsolute = setting === path.resolve(setting);
+			m.chai.expect(isAbsolute).to.be.true;
+		}));
+
+	describe('.projectsDirectory', () =>
+		it('should be an absolute path', () => {
+			const setting = utils.evaluateSetting<string>(
+				defaults,
+				'projectsDirectory'
+			);
+			const isAbsolute = setting === path.resolve(setting);
+			m.chai.expect(isAbsolute).to.be.true;
+		}));
+
+	describe('.cacheDirectory', () =>
+		it('should be an absolute path', () => {
+			const setting = utils.evaluateSetting<string>(defaults, 'cacheDirectory');
+			const isAbsolute = setting === path.resolve(setting);
+			m.chai.expect(isAbsolute).to.be.true;
+		}));
+
+	describe('.imageCacheTime', () => {
+		it('should be a number', () => {
+			const setting = utils.evaluateSetting<number>(defaults, 'imageCacheTime');
+			m.chai.expect(setting).to.be.a('number');
+		});
+
+		it('should be a positive number', () => {
+			const setting = utils.evaluateSetting<number>(defaults, 'imageCacheTime');
+			m.chai.expect(setting > 0).to.be.true;
```

Comment:
`expect(setting).to.be.greaterThan(0)`

Avoids the unused expression syntax, and makes errors way clearer ('-100 was less than 0', not just 'false != true')

I know these are preexisting, but since we're here and its super quick, lets make them nice.